expr/data_test.go

```diff
@@ -312,8 +312,18 @@ func getCopy(in []schema.Point) []schema.Point {
 	return out
 }
 
+func getSeries(target, patt string, data []schema.Point) models.Series {
+	return models.Series{
+		Target:     target,
+		QueryPatt:  patt,
+		Datapoints: getCopy(data),
+		Interval:   10,
+	}
+}
+
 func getQuerySeries(patt string, data []schema.Point) models.Series {
```

Comment:
getQuerySeries and getTargetSeries are now equivalent. perhaps we should consolidate?